bounties/npm/cordova-serve/1/bounty.json

```diff
@@ -0,0 +1,7 @@
+{
+    "ForkURL": "https://github.com/bitovi/launchpad/",
```

Comment:
This is in the cordora-serve directory but the fork url is for bitovi launchpad?